ptf_tests/p4ovs_test_runner.py

```diff
@@ -149,28 +149,43 @@ def print_help(self):
             if i.startswith("#"):
                 continue
             items = i.strip().split(":")
-            test_to_run[items[0].strip()] = ':'.join(items[1:]).strip()
-            sequence.append(items[0])
+            scriptname = items[0].strip()
+
+            # if there are tests with same script name but different parameters (json file etc.)
+            occurence = len([1 for x in test_to_run.keys() if re.match(scriptname,x)])
```

Comment:
It will only works for two same script. If having 3 or more same script with different parameter, the "occurence"  will be always 1. Then 3rd and so on will be executed by scriptname_1 instead. My understanding is

```
with open(args.file) as fh:
       occurence=0
        for i in fh.readlines():
           .......
           if scriptname in test_to_run.keys():
               occurence +=1
               scriptname = scriptname + f"_{occurence}"
```

Reply:
I have verified this, please check:

import re
l = ['test', 'test_1','test_2','test_3']
scriptname='test'
occurence = len([1 for x in l if re.match(scriptname,x)])
print(occurence)
4